app/editor/src/features/admin/reports/ReportSections.tsx

```diff
@@ -0,0 +1,134 @@
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { FaArrowDown, FaArrowUp, FaTrash } from 'react-icons/fa';
+import { Button, ButtonVariant, Col, IReportModel, Row } from 'tno-core';
+
+import { IReportSection } from './interfaces';
+import { ReportSection } from './ReportSection';
+import * as styled from './styled';
+
+const defaultSection = (sortOrder: number): IReportSection => {
+  return {
+    name: '',
+    label: '',
+    description: '',
+    sortOrder: sortOrder,
+    isEnabled: true,
+    filter: {},
+  };
+};
+
+export const ReportSections = () => {
```

Comment:
I need to add validation to ensure each section has a unique name.  But I'll leave it for now as this is only for administrators presently.